docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,47 @@ lang: en
 
 ### To update the plugin, please follow the linked [instructions](./en/getting-started/install/#upgrade-plug-in)
 
+## [v2.13.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.13.0) (02-23-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.12.0...v2.13.0)
+
+### Release Summary
+
+* Updated PMD to 6.42.0
+	* Note: EagerlyLoadedSObjectDescribeResult rule is skipped
+* Upgrades to local RetireJS Vulnerability Repository
+
+
+**Closed issues:**
+
+- \[BUG\] Cannot find Config.json after installing sfdx-scanner in an Ubuntu machine [\#581](https://github.com/forcedotcom/sfdx-scanner/issues/581)
+- \[BUG\] JavaScript heap out of memory while invoking RetireJS engine [\#570](https://github.com/forcedotcom/sfdx-scanner/issues/570)
+- \[BUG\] [\#569](https://github.com/forcedotcom/sfdx-scanner/issues/569)
+- \[RESOLVED\] Salesforce Scanner CLI Plugin - Heap Out of Memory Issue [\#564](https://github.com/forcedotcom/sfdx-scanner/issues/564)
+- \[Feature Request\] Compatibility with lint-staged [\#551](https://github.com/forcedotcom/sfdx-scanner/issues/551)
+- Windows install results in "Invalid Character" Windows Host Script error [\#539](https://github.com/forcedotcom/sfdx-scanner/issues/539)
+- \[BUG\] Unable to install sfdx-scanner: FailedDigitalSignatureVerification [\#453](https://github.com/forcedotcom/sfdx-scanner/issues/453)
+
+**Merged pull requests:**
+
+- @W-10731711@: Updated RetireJS to latest version. [\#583](https://github.com/forcedotcom/sfdx-scanner/pull/583)
+- @W-8885529@ Excluding transitive dependency on junit and hamcrest [\#582](https://github.com/forcedotcom/sfdx-scanner/pull/582)
+- @W-10477402@ Recreate symlinks to work around nvm issue [\#580](https://github.com/forcedotcom/sfdx-scanner/pull/580)
+- @W-10667880@: Updated documentation to include information about standalone XML files. [\#578](https://github.com/forcedotcom/sfdx-scanner/pull/578)
+- @W-10530556@: Updated locally-stored RetireJS vulnerability catalog. [\#577](https://github.com/forcedotcom/sfdx-scanner/pull/577)
+- @W-10488971@: Upgrading PMD from 6.38.0 to 6.42.0. [\#576](https://github.com/forcedotcom/sfdx-scanner/pull/576)
+- @W-10488954@: Implemented hardcoded rule skipping mechanism. [\#574](https://github.com/forcedotcom/sfdx-scanner/pull/574)
+- @W-10477617@: Switched to a circleci context for publishing and signing. [\#573](https://github.com/forcedotcom/sfdx-scanner/pull/573)
```

Comment:
573, 568, 566, 562, 559, 557, 556, and 548 probably don't need to be included in the release notes.